### PR TITLE
[java] improve binding for lsp-java

### PR DESCRIPTION
**Pull request by thanhvg**
just some small improvement to java layer. Removed some obsolete commands and added some. Tag @yyoncho for review

### CODE REVIEW

File: layers/+lang/java/README.org [hunk 1]
```diff
@@ -117,9 +117,12 @@ at the root of your project first.
 | ~SPC m g d~   | Goto type definition                     |
 | ~SPC m g a~   | Search type in project                   |
 | ~SPC m g A~   | Search type in project using ~lsp-ui~    |
+| ~SPC m g k k~ | Show type hierarchy                      |
```
Comment (yyoncho) [inline layers/+lang/java/README.org:120]: tb is missing

**Review by yyoncho — APPROVED** [review]

File: layers/+lang/java/README.org (hunk 1 above)
Comment (yyoncho) [reply layers/+lang/java/README.org:120]: Also IMHO, type hierarchy should be under `,gh`

Comment (thanhvg) [reply layers/+lang/java/README.org:120]: will add `t b`, `,gh` is used by call hierarchy

Comment (yyoncho) [reply layers/+lang/java/README.org:120]: then `,gH`?

Comment (thanhvg) [reply layers/+lang/java/README.org:120]: I'd prefer lower case letters. imo bindings having upper case letter is a pain to invoke.

**Review by smile13241324 — APPROVED** [review]
Looks good, I will merge it in a minute.

**Pull request merged**